core/templates/dev/head/services/GenerateContentIdServiceSpec.ts

```diff
@@ -60,6 +44,7 @@ describe('GenerateContentIdService', function() {
 
   it('should throw error for unknown content id', function() {
```

Comment:
Please change the frontend tests from `function() {` to `() => {`

Reply:
Done